### PR TITLE
cloudproviders: add ovirt getting started guide

### PR DESCRIPTION
**Pull request by simon3z**

### CODE REVIEW

File: docs/getting-started-guides/ovirt.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## What is oVirt
+
+oVirt is a virtual datacenter manager that delivers powerful management of multiple virtual machines on multiple hosts. Using KVM and libvirt, oVirt can be installed on Fedora, CentOS, or Red Hat Enterprise Linux hosts to set up and manage your virtual data center.
+
+## oVirt Cloud Provider Deployment
+
+The oVirt cloud provider allows to easily discover and automatically add new VM instances as nodes to you kubernetes cluster.
```
Comment (a-robinson) [inline docs/getting-started-guides/ovirt.md]: s/you/your

File: docs/getting-started-guides/ovirt.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+At the moment there are no community-supported or pre-loaded images including kubernetes but it is possible to import or install Project Atomic in a VM to generate a template. Any other distribution that includes kubernetes may work as well.
```
Comment (a-robinson) [inline docs/getting-started-guides/ovirt.md]: Can you clarify what you mean by images? OS/VM images or Docker images?

What does it mean to "generate a template"? What's it a template of or for?

File: docs/getting-started-guides/ovirt.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+It is mandatory to install the ovirt-guest-agent in the guests for the VM ip address and hostname to be reported to ovirt-engine and ultimately to kubernetes.
```
Comment (a-robinson) [inline docs/getting-started-guides/ovirt.md]: How?

File: docs/getting-started-guides/ovirt.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Once the kubernetes template is available it is possible to start instantiating VMs that can be discovered by the cloud provider.
+
+## Using the oVirt Cloud Provider
+
+The oVirt Cloud Provider requires to access the oVirt REST-API to gather the proper information, the required credential should be specified in the `ovirt-cloud.conf` file:
```
Comment (a-robinson) [inline docs/getting-started-guides/ovirt.md]: s/requires to access/requires access to

File: docs/getting-started-guides/ovirt.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+In the above example all the VMs tagged with the `kubernetes` label will be reported as nodes to kubernetes.
+
+The `ovirt-cloud.conf` file then must be specified in kube-controller-manager:
+
+    kube-controller-manager ... --cloud-provider=ovirt --cloud-config=/path/to/ovirt-cloud.conf ...
+
+## oVirt Cloud Provider Screencast
+
+This short screencast demostrates how the oVirt Cloud Provider can be used to dynamically add VMs to your kubernetes cluster.
```
Comment (a-robinson) [inline docs/getting-started-guides/ovirt.md]: s/demostrates/demonstrates

**Comment by a-robinson** [comment]
Thanks for circling back around to this, just a few minor fixes and clarifications before I'll merge.

**Comment by simon3z** [comment]
@a-robinson this should be fine now.

**Comment by a-robinson** [comment]
LGTM

**Pull request merged**